ui/src/containers/Search/index.tsx

```diff
@@ -1,19 +1,39 @@
 import { TextInput } from '@patternfly/react-core';
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useMst } from '../../store/root';
+import { UseDebounce } from './useDebounce';
 import './Search.css';
 
 const Search: React.FC = () => {
   const { resources } = useMst();
-
   const [value, setValue] = useState('');
+  let url = '';
+
+  // to get query param from thye url
+  const searchParams = new URLSearchParams(window.location.search);
```

Comment:
What does this `window.location.search` do :thinking: ??

Reply:
it finds query params from url

Reply:
How does it gets cleared, are we storing it in localstorage and if someone's open with the query params will we get the expected result

Reply:
we are not storing it in localstorage , yeah will get the same result